src/wallet-web-api-swagger/Main.hs

```diff
@@ -1,43 +1,39 @@
-{-# LANGUAGE FlexibleInstances   #-}
 {-# LANGUAGE KindSignatures      #-}
-{-# LANGUAGE OverloadedStrings   #-}
 {-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE FlexibleInstances   #-}
```

Comment:
I feel that stylish-haskell wasn't applied to this file – can you apply it? (with standard Serokell settings)